easybuild/easyblocks/t/tensorflow.py

```diff
@@ -638,13 +634,9 @@ def configure_step(self):
         if LooseVersion(self.version) < LooseVersion('1.12.0'):
             # patch configure.py (called by configure script) to avoid that Bazel abuses $HOME/.cache/bazel
             regex_subs = [(r"(run_shell\(\['bazel')",
-                           r"\1, '--output_base=%s', '--install_base=%s'" % (self.output_base_dir,
-                                                                             self.install_base_dir))]
+                           r"\1, '--output_user_root=%s'" % self.output_user_root_dir)]
             apply_regex_substitutions('configure.py', regex_subs)
 
-        # Tell Bazel to not use $HOME/.cache/bazel at all
-        # See https://docs.bazel.build/versions/master/output_directories.html
-        env.setvar('TEST_TMPDIR', self.output_root_dir)
```

Comment:
@Flamefire Not setting this results in Bazel using `$HOME/.cache/bazel` again...

For `TensorFlow-1.14.0-foss-2019a-Python-3.7.2.eb` for example, after running `touch $HOME/.cache/bazel; chmod 000 $HOME/.cache/bazel`:

```
== FAILED: Installation ended unsuccessfully (build directory: /tmp/easybuild_build/TensorFlow/1.14.0/foss-2019a-Python-3.7.2): build failed (first 300 chars): cmd "./configure " exited with exit code 1 and output:
FATAL: mkdir('/home/kehoste/.cache/bazel/_bazel_vsc40023'): (error: 20): Not a directory
```

Reply:
Oh, thanks! Will check and refine as from what I could gather this shouldn't happen.
Point here is (to my understanding): This variable is to be **used** by tests, not set by the user and I have found code in Bazel which will make it run "differently" when it detects that it is a test instance by checking this variable for being set (yeah... I find that "interesting" too)

Reply:
Ok found the problem: This is caused by the configure process which runs bazel directly. We had an easyblock patch for that but that didn't apply after 1.12 hence was removed for that versions.
If you redo your test with the current easyblock I expect the same behavior.
My suggestion is to merge https://github.com/easybuilders/easybuild-framework/pull/3440 and make sure the configure patch applies via `on_missing_match='error'`

Reply:
Should be fixed by 530ca095
See also https://github.com/tensorflow/tensorflow/commit/56a99d6c28fd0ff238bd834bd32c2d3fc894b86c for the max version

Reply:
OK, looks good after testing (`~/.cache/bazel` stays empty)